composer.json

```
@@ -17,7 +17,7 @@
         "laravel/sail": "^1.41",
         "mockery/mockery": "^1.6",
         "nunomaduro/collision": "^8.6",
-        "phpunit/phpunit": "^11.5.3"
+        "phpunit/phpunit": "^12.1.2"
```

PHPUnit 12 requires PHP 8.3 as minimum

---

```suggestion
        "phpunit/phpunit": "^11.5.3 || ^12.1.2"
```

@crynobone Yeah, I just saw that too.

Should it be kept at PHPUnit 11 in the skeleton, or allow both?